simmechanics/data/hand_mk3/simmechanics2urdf_configfile.yaml

```diff
@@ -0,0 +1,106 @@
+# Meshes options 
+scale: "0.001 0.001 0.001"
+forcelowercase: Yes
+filenameformatchangeext: "package://iCub/meshes/simmechanics/%s.stl"
+stringToRemoveFromMeshFileName: ""
+
+# Rename options (map Creo names to URDF names)
+rename:
+ SIM_L_HAND_PALM_NAKED--SIM_L_HAND_PALM_ADDUCTION: JOINT-palm
+ SIM_L_HAND_PALM_ADDUCTION--SIM_L_HAND_METACARPUS_THUMB: JOINT-metacarpus_thumb
+ SIM_L_HAND_METACARPUS_THUMB--SIM_L_HAND_PROXIMAL_THUMB: JOINT-prox_thumb
+ SIM_L_HAND_PROXIMAL_THUMB--SIM_L_HAND_DISTAL_THUMB: JOINT-distal_thumb
+ SIM_L_HAND_PALM_NAKED--SIM_L_HAND_METACARPUS_INDEX: JOINT-metacarpus_index
+ SIM_L_HAND_METACARPUS_INDEX--SIM_L_HAND_PROX_INDEX_FINGER: JOINT-prox_index
+ SIM_L_HAND_PROX_INDEX_FINGER--SIM_L_HAND_DISTAL_INDEX_FINGER: JOINT-distal_index
+ SIM_L_HAND_PALM_NAKED--SIM_L_HAND_PROX_MEDIUM_FINGER: JOINT-prox_medium
+ SIM_L_HAND_PROX_MEDIUM_FINGER--SIM_L_HAND_DISTAL_MEDIUM_FINGER: JOINT-distal_medium
+ SIM_L_HAND_PALM_NAKED--SIM_L_HAND_PROX_LITTLE_FINGER: JOINT-prox_littlefinger
+ SIM_L_HAND_PROX_LITTLE_FINGER--SIM_L_HAND_DISTAL_LITTLE_FINGER: JOINT-distal_littlefinger
+ SIM_L_HAND_PALM_NAKED: l_hand_palm
+ SIM_L_HAND_PALM_ADDUCTION: l_hand_palm_adduction 
+ SIM_L_HAND_METACARPUS_THUMB: l_hand_metacarpus_thumb 
+ SIM_L_HAND_PROXIMAL_THUMB: l_hand_proximal_thumb 
+ SIM_L_HAND_DISTAL_THUMB: l_hand_distal_thumb
+ SIM_L_HAND_METACARPUS_INDEX: l_hand_metacarpus_index 
+ SIM_L_HAND_PROX_INDEX_FINGER: l_hand_prox_index_finger 
+ SIM_L_HAND_DISTAL_INDEX_FINGER: l_hand_distal_index_finger
+ SIM_L_HAND_PROX_MEDIUM_FINGER: l_hand_prox_medium_finger 
+ SIM_L_HAND_DISTAL_MEDIUM_FINGER: l_hand_distal_medium_finger 
+ SIM_L_HAND_PROX_LITTLE_FINGER: l_hand_prox_little_finger 
+ SIM_L_HAND_DISTAL_LITTLE_FINGER: l_hand_distal_little_finger 
```

Comment:
Are these joint names documented somewhere? On iCub 2 we have exactly this problem that the joint names of the hand are not defined (see https://github.com/robotology/icub-models/issues/30), it would be great if we could start documenting the joint names for the hand of iCub3, do avoid this problem (see for example https://icub-tech-iit.github.io/documentation/icub_kinematics/icub-joints/icub-joints/).

Reply:
To given an example why this is important, it seems that the configuration files in https://github.com/robotology/robots-configuration/pull/253/files#diff-0fb41e3c08177cef454c0dc1e823aac1a46346e4d44c58a008dd2912f67131d0R12 use completely different joint names, or perhaps the relation between this two set of names is not clear.

Reply:
cc @pattacini